scripts/e2e/issue151_selection_linux.py

```
@@ -0,0 +1,201 @@
+import pathlib
+import subprocess
+import sys
+import time
+
+MARKER = "ISSUE151_ALPHA"
+WINDOW_X = 0
+WINDOW_Y = 0
+
+
+def run(*args: str, check: bool = True, capture: bool = True) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(args, check=check, text=True, capture_output=capture)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)